### PR TITLE
Final 1 0

### PR DESCRIPTION
**Pull request by titusfortner**
this fixes & combines #152 & #153 
I was going to add some other things, but I think this is sufficient for 1.0

### CODE REVIEW

**Review by nadvolod — APPROVED** [review]

File: java/src/main/java/com/saucelabs/saucebindings/SaucePlatform.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import java.util.Set;
 
 public enum SaucePlatform {
+    LINUX("Linux"),
```
Comment (joshmgrant) [inline java/src/main/java/com/saucelabs/saucebindings/SaucePlatform.java:10]: We have to do this, don't we @titusfortner

**Review by joshmgrant — COMMENTED** [review]
Looks good except for the LINUX platform.

File: java/src/main/java/com/saucelabs/saucebindings/SaucePlatform.java (hunk 1 above)
Comment (titusfortner) [reply java/src/main/java/com/saucelabs/saucebindings/SaucePlatform.java:10]: Yeah, it's required for Headless, unless we want to handle headless differently via Session (which I'm also ok with)

Comment (titusfortner) [reply java/src/main/java/com/saucelabs/saucebindings/SaucePlatform.java:10]: I feel like we either have to add this now or remove US_EAST support from Session.

Comment (joshmgrant) [reply java/src/main/java/com/saucelabs/saucebindings/SaucePlatform.java:10]: Nah let's keep it like this for now.

**Review by joshmgrant — APPROVED** [review]

**Pull request merged**